rpc/answer.go

```diff
@@ -206,18 +202,17 @@ func (ans *answer) Return(e error) {
 // Finish with releaseResultCaps set to true, then sendReturn returns
 // the number of references to be subtracted from each export.
 //
-// The caller must be holding onto ans.c.mu and the sender lock.
-// The result's capability table must have been extracted into
-// ans.resultCapTable before calling sendReturn. Only one of
-// sendReturn or sendException should be called.
+// The caller MUST hold ans.c.mu.  The result's capability table MUST
+// have been extracted into ans.resultCapTable before calling sendReturn,
+// and call only one of sendReturn or sendException.
```

Comment:
>Only one of sendReturn or sendException should be called

This can be interpreted as a recommendation rather than the hard requirement that _either_ `sendReturn` or `sendException` _must_ be called.

How do you feel about the following?

>The caller MUST be holding onto ans.c.mu.  The result's capability table
MUST have been extracted into ans.resultCapTable before calling sendReturn.
sendReturn MUST NOT be called if sendException was previously called.

Reply:
SGTM.